Sources/NMeta/Extensions/Application+NMeta.swift

```diff
@@ -1,11 +1,8 @@
 import Vapor
 
-private struct NMetaKey: StorageKey {
```

Comment:
@siemensikkema  Just a thought, isn't more clear this struct to stay outside in the beginning of the file as it was, but to be `fileprivate` instead to be nested? 🤔

Reply:
@emarashliev I went with what I saw was being used inside Vapor itself. I don't know, I kind of like using the nesting like this ... Maybe I'll reconsider later 🙂 luckily it doesn't matter for the public API